.github/workflows/checks.yml

```diff
@@ -13,9 +13,6 @@ concurrency:
   group: checks-${{ github.ref }}
   cancel-in-progress: true
 
-env:
-  NODE_OPTIONS: --max_old_space_size=8192
-
```

Comment:
I'd keep this as the CI config. If this is ever modified, the script would not override it.